### ISSUE TITLE
Accessibility checks during build process

### ISSUE THREAD

**Issue by robmcguinness** [Availity/availity-uikit#176]

**Comment by robmcguinness** [comment]
Something like: 

```js
const axe = require('axe-core');

  describe('Custom component', function() {
    it('should have no a11y violations', function(done) {
      let options = {};

      axe.run('.some-element-selector', options, (error, results) => {
        expect(results.violations.length).toBe(0);
        done();
      });
    });
  });
```

**Comment by christieb7007** [comment]
not sure if we can get the html passed into the `axe.run` test, but `gatsby-plugin-accessibilityjs` looks like a good inline alternative. Errors found once the site is running are automatically logged in the console and can be configured to highlight or not.
